packages/core/src/components/widgets/CheckboxWidget.tsx

```diff
@@ -1,22 +1,20 @@
 import React from "react";
-import PropTypes from "prop-types";
 import { schemaRequiresTrueValue } from "@rjsf/utils";
+import { WidgetProps } from "@rjsf/utils";
 
-function CheckboxWidget(props) {
-  const {
-    schema,
-    id,
-    value,
-    disabled,
-    readonly,
-    label,
-    autofocus,
-    onBlur,
-    onFocus,
-    onChange,
-    registry,
-  } = props;
-
+function CheckboxWidget<T = any, F = any>({
```

Comment:
Can you add JSDoc here as well... take a look at how the other, already converted widgets are documented as a guide